service/controller/app/v1/resource/chart/delete.go

```diff
@@ -32,7 +26,7 @@ func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange inte
 			return microerror.Mask(err)
 		}
 
-		err = cc.G8sClient.ApplicationV1alpha1().Charts(cr.GetNamespace()).Delete(chart.Name, &metav1.DeleteOptions{})
+		err = cc.G8sClient.ApplicationV1alpha1().Charts(r.chartNamespace).Delete(chart.Name, &metav1.DeleteOptions{})
```

Comment:
Can it be following?

```suggestion
		err = cc.G8sClient.ApplicationV1alpha1().Charts(chart.Namespace).Delete(chart.Name, &metav1.DeleteOptions{})
```

Reply:
Well that's same but looks nicer than before :)